src/components/Category/Category.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import styled from 'styled-components'
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { useParams } from "react-router-dom";
+
+import categoryTemp from 'assets/categoryTemp.jpeg'
+
+import Image from './components/Image'
+
+
+const dummyData = [
+    {
+        title: 'XDD',
+        date: '21.21.2002'
+    },
+    {
+        title: 'XDD',
+        date: '21.21.2002'
+    },
+    {
+        title: 'XDD',
+        date: '21.21.2002'
+    },
+]
+
+const StyledContainer = styled(Container)`
+    margin:0;
+    padding: 0!important;
+`;
+
+const Category = () => {
+    const [categories, setCategories] = useState([])
+    const params = useParams();
+
+    const reciveData = () => {
+        console.log(params)
+        fetch('https://example.com/profile', {
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        })
+            .then(response => response.json())
+            .then(data => {
+                console.log('Success:', data);
+                setCategories(data)
+                console.log(categories)
+            })
+            .catch((error) => {
+                console.error('Error:', error);
+            });
+    }
```

Comment:
tego fetcha dałbym do oddzielnego pliku i console.logi pousuwał (albo ogólnie z tego pr ten plik usunął w sumie)

Reply:
To w sumie usune bo to faktycznie tu bez sensu